packages/ibm-products/src/components/CreateTearsheet/CreateTearsheet.test.js

```diff
@@ -79,6 +82,10 @@ const renderCreateTearsheet = ({
   onPrevious = onPreviousStepFn,
   finalOnNextFn = finalStepOnNext,
   rejectOnSubmitNext = false,
+  isCustomButtonDisabled = false,
+  isCustomButtonHide = false,
```

Comment:
for these i think it's better to go with the opposite naming convention. so like `customButtonEnabled` and `customButtonVisible`

Reply:
I think this one probably should be `customButtonDisabled` since it’s about the disabled state. 🤔